mlflow/tracing/display/display_handler.py

```diff
@@ -21,6 +72,36 @@ def _serialize_trace_list(traces: list["Trace"]):
     )
 
 
+def _get_query_string_for_traces(traces: list["Trace"]):
+    query_params = []
+
+    for trace in traces:
+        query_params.append(("trace_id", trace.info.request_id))
+        query_params.append(("experiment_id", trace.info.experiment_id))
+
+    return urlencode(query_params)
+
+
+def is_jupyter():
+    try:
+        from IPython import get_ipython
+
+        return get_ipython() is not None
+    except ImportError:
+        return False
+
+
+def is_using_tracking_server():
+    return is_http_uri(mlflow.get_tracking_uri())
+
+
+def validate_environment():
```

Comment:
Maybe `is_trace_ui_available`?

I expect a method to raise an exception if the function name contains "validate".